pnas/PNAS-template-main.tex

```diff
@@ -291,20 +288,20 @@ \subsection*{Best response memory-one strategies}
 \(q=(1, 1, 1, 1)\), is a defector \(p^*=(0, 0, 0, 0)\). The strategies
 \(p=(\frac{1}{2}, 0, 0, 0)\) and \(p=(\frac{1}{2}, 0, 0, \frac{1}{2})\) are also
 best responses. The approach taken here is to introduce a compact way of
-constructing the candidate set of all local optimal points, and evaluating which
+constructing the discrete candidate set of all local optimal points, and evaluating which
 corresponds to the best response strategy. The approach is given in Theorem~\ref{memone_group_best_response}.
 
 Finding best response memory-one strategies is analytically feasible using the
 formulation of Theorem~\ref{memone_group_best_response} and resultant
 theory~\cite{Jonsson2005}. However, for large systems building the resultant
-quickly becomes intractable. As a result, best responses will be estimated
-heuristically using a numerical method taking advantage of the structure called
+becomes intractable. As a result, best responses will be estimated
+heuristically using a numerical method taking advantage of the technique called
```

Comment:
`a technique`

Reply:
let's chat.

Reply:
I think the technique does not refer to Bayesian optimisation but our approach discussed above.

Reply:
I don't think that fully reads right to me then it might need a comma somewhere...

Reply:
```
As a result, best responses will be estimated
heuristically using a numerical method called Bayesian optimisation~\cite{Mokus1978}.
```

or

```
As a result, best responses will be estimated
heuristically using a numerical method, suitable for problems with local optima, called Bayesian optimisation~\cite{Mokus1978}.
```